charts/jenkins/Chart.yaml

```diff
@@ -35,7 +35,7 @@ annotations:
       url: https://github.com/jenkinsci/helm-charts/issues
   artifacthub.io/images: |
     - name: jenkins
-      image: docker.io/jenkins/jenkins:2.426.3-jdk17
+      image: docker.io/jenkins/jenkins:2.440.1-windowsservercore-ltsc2019
```

Comment:
it's updated it to the windows version =/

Reply:
I'll try to look into this this evening. For now it should be easy enough to make a slight change to #1020 instead.